report-job-status/action.yml

```diff
@@ -32,6 +39,19 @@ runs:
           ${{ github.event.head_commit.message }}
       shell: bash
 
+    - name: Get job data
+      id: get-job-data
+      shell: bash
+      env:
+        CURL_COMMAND: "curl --location --fail --silent --show-error --retry 5 --retry-delay 5 "
+        API_ROOT: https://api.github.com/repos/${{ github.repository }}/actions/runs
```

Comment:
You can use this variant to fit it to 80 chars:
```
        API_ROOT: >-
          https://api.github.com/repos/${{ github.repository }}/actions/runs
```

Reply:
fixed